### DISCUSSION TITLE
Accounting user

### DISCUSSION THREAD

**Discussion by ctpntn** [kimai/kimai#2313] [Questions & Answers]
I'm playing with permissions. I'd like to get an accounting user who can see all users' records and download them in an excel file, but who is not be able to put them in cleared status. It seems that only teamleaders have this possibility.

**Comment by ctpntn** [comment]
So is there no way to ONLY extract data into Excel format?

**Reply by ctpntn** [reply]
In few words I need that some users can use this men
<img width="183" alt="Schermata 2021-02-08 alle 11 20 33" src="https://user-images.githubusercontent.com/3852346/107206769-c5bce500-69ff-11eb-98fe-25392b9be04b.png">
in order to export data, but they have not to be able Export/Clear record.

**Reply by kevinpapst** [reply]
Play with different sets of permissions. Nobody can help you with that. I have never tried it. You might be able to achieve it with a combination of a team and permissions. What about `export_other_timesheet`?

**Reply by kevinpapst** [reply]
Activate `view_other_timesheet`, deactivate `edit_other_timesheet`. You should use a new role for that.

**Reply by ctpntn** [reply]
It was simple! Thank you
